src/browser/components/MainPage.jsx

```diff
@@ -89,22 +95,40 @@ export default class MainPage extends React.Component {
     return remote.webContents.getAllWebContents().find((webContents) => webContents.getURL().includes(tabURL));
   }
 
+  focusListener() {
+    this.handleOnTeamFocused(this.state.key);
+    if (this.refs[`mattermostView${this.state.key}`]) {
+      this.refs[`mattermostView${this.state.key}`].focusOnWebView();
+    }
+  }
+
   componentDidMount() {
-    const self = this;
```

Comment:
Why?
Cleaner and unnecessary.  Either lexically bind functions with arrow functions or `bind` scope to functions.